### PR TITLE
400 * ln(14400 * 100) * 6 / 60 ~= 567

### PR DESCRIPTION
**Pull request by pampatzoglou**
closes https://github.com/w3f/infrastructure/issues/231

400 * ln(14400 * 100) * 6 / 60 ~= 567
400 * ln(14400 * 1000) * 6 / 60 ~= 659

### CODE REVIEW

File: charts/polkadot-watcher/templates/alertrules.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ spec:
         message: 'Blocks were not produced for 5:19 hours by {{`{{ $labels.name }}`}}'
```
Comment (fgimenez) [inline charts/polkadot-watcher/templates/alertrules.yaml:14]: We need to update the message too

File: charts/polkadot-watcher/templates/alertrules.yaml [hunk 2]
```diff
@@ -23,7 +23,7 @@ spec:
         message: 'Blocks were not produced for 6:11 hours by {{`{{ $labels.name }}`}}'
```
Comment (fgimenez) [inline charts/polkadot-watcher/templates/alertrules.yaml:23]: Same the message should be updated to match the new value.

File: charts/polkadot-watcher/templates/alertrules.yaml (hunk 1 above)
Comment (pampatzoglou) [reply charts/polkadot-watcher/templates/alertrules.yaml]: updated thx

File: charts/polkadot-watcher/templates/alertrules.yaml (hunk 2 above)
Comment (pampatzoglou) [reply charts/polkadot-watcher/templates/alertrules.yaml]: updated

**Pull request merged**